addons/web/static/src/core/record_selectors/record_selector.xml

```diff
@@ -2,18 +2,21 @@
 <templates xml:space="preserve">
 
     <t t-name="web.RecordSelector" >
-        <RecordAutocomplete
-            resModel="props.resModel"
-            value="displayName"
-            domain="props.domain"
-            context="props.context"
-            className="'h-100 o_record_selector'"
-            fieldString="props.fieldString"
-            placeholder="props.placeholder"
-            multiSelect="false"
-            getIds="() => []"
-            update.bind="update"
-        />
+        <div class="o_input d-flex o_record_selectors_dropdown">
```

Comment:
The class o_record_selectors_dropdown is weird. Can you change it for o_record_selector maybe?
I also note that this div is missing some bootstrap classes that are set in the template "web.MultiRecordSelector".
Is it normal? have you checked that it is fine to add a div here? RecordSelector is used in many places?

Reply:
Hello,

the class `o_record_selectors_dropdown` has been removed. Instead of it I have added `o_record_selector`.
`RecordSelector` is used mostly in `web_studio` and with the `div` it works fine. Also at those places the caret was not visible  though recordselector was there. But now after this patch whereever Recordselector is present the caret is visible on hover as per the expected flow.

Please let me know if I'm overlooking anything.

Thanks